### PR TITLE
Support changing loss, fix #86

### LINKED ISSUE #86: [feature] modify loss
By default,

```python
        loss = nn.CrossEntropyLoss()
```

Can it be changed to other function to make this package more feasible and customizable?

### PR DESCRIPTION
**Pull request by Freed-Wu**
## PR Type and Checklist
What kind of change does this PR introduce?

- [ ] FIX BUG
	- [ ] Check CURRENT and PREVIOUS [ISSUES](https://github.com/Harry24k/adversarial-attacks-pytorch/issues)..
	- [ ] Check [WORFLOW](https://github.com/Harry24k/adversarial-attacks-pytorch/actions) result  after PR.
- [ ] ADD ATTACK
	- [ ] DEFAULT values of arguments should be given except `model`.
	- [ ] Classname should be given as CamelCase.
	- [ ] Python file name should be SMALL letters of the classname.
	- [ ] Add attack in [__init__.py](https://github.com/Harry24k/adversarial-attacks-pytorch/blob/master/torchattacks/__init__.py)
	- [ ] Check `supported_mode` wheter the attack supports targeted mode.
	- [ ] [README.md](https://github.com/Harry24k/adversarial-attacks-pytorch/blob/master/README.md)  should be updated.
	- [ ] Check [WORFLOW](https://github.com/Harry24k/adversarial-attacks-pytorch/actions) result  after PR.
- [x] Other... Please describe: fix #86

### CODE REVIEW

**Comment by Harry24k** [comment]
Thank you for your contribution!
However, as I mentioned in https://github.com/Harry24k/adversarial-attacks-pytorch/issues/86, the loss function is highly correlated with the attack. For example, the changing loss can yield a new version of an attack such as APGD.
Thus, I think the changing loss can be only added to UPGD, which is an integrated version of attacks.
